Checkout/Sample/Source/UIColor+CKO.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UIColor+CKO.swift
+//
+//
+//  Created by Harry Brown on 28/02/2022.
+//
+
+import UIKit
+
+extension UIColor {
+  static let ckoDarkBlue = UIColor(
+    red: 12 / 255,
+    green: 17 / 255,
+    blue: 66 / 255,
+    alpha: 1
+  )
+
+  static let ckoLightBlue = UIColor(
+    red: 41 / 255,
+    green: 212 / 255,
+    blue: 219 / 255,
+    alpha: 1
+  )
+
+  static let ckoLightYellow = UIColor(
```

Comment:
I think there were some changes introduced by @ehab-al-cko , could we reuse the fame file?

Reply:
Wouldn't be available in this context, those colours are in the Frames SDK